src/@types/error-message-types.ts

```diff
@@ -15,15 +7,19 @@ export enum ErrorMessageType {
   INVALID_CARD_TYPE = 'INVALID_CARD_TYPE',
   INVALID_CARD_DETAILS = 'INVALID_CARD_DETAILS',
   SERVER_ERROR = 'SERVER_ERROR',
-  ALREADY_PROCESSED = 'ALREADY_PROCESSED',
   TIMEOUT = 'TIMEOUT',
-  INVALID_ACTION = 'INVALID_ACTION',
   NO_PAY_SECRET = 'NO_PAY_SECRET',
   UNKNOWN_ERROR = 'UNKNOWN_ERROR',
-  ENVIRONMENT_MISMATCH = 'ENVIRONMENT_MISMATCH',
   FAILED_TO_SUBMIT = 'FAILED_TO_SUBMIT',
 }
 
+export enum ErrorCodes {
+  PAY_REQUEST_INVALID_STATUS = 'PAY_REQUEST_INVALID_STATUS',
+  ENVIRONMENT_MISMATCH = 'ENVIRONMENT_MISMATCH',
+  PAY_REQUEST_SECRET_REQUIRED = 'PAY_REQUEST_SECRET_REQUIRED',
+  WALLET_INIT_FAILED = 'WALLET_INIT_FAILED',
```

Comment:
Some of these codes are not listed in the docs, are they to be added? Also these are errorCodes but errorTypes in the docs?

And there appears to be like, "NO_PAY_SECRET" in the docs, but here its mentioned as PAY_REQUEST_SECRET_REQUIRED

Reply:
yes to be added. Also I've tried to reuse existing errorCodes if they overlapped a little like with PAY_REQUEST_INVALID_STATUS and PAY_REQUEST_SECRET_REQUIRED (formerly the NO_PAY_SECRET type) while intending to add ENVIRONMENT_MISMATCH and WALLET_INIT_FAILED as new codes.

Thoughts on this?

Reply:
Oka, main thing was I couldn't see them in your docs branch.

I still dont like errorTypes.....